src/DynamoPackages/PackageLoader.cs

```diff
@@ -411,7 +418,7 @@ public void LoadCustomNodesAndPackages(IEnumerable<string> newPaths, IPreference
             if (LocalPackages.Any())
             {
                 // Load only those recently addeed local packages (that are located in any of the new paths)
```

Comment:
this comment now seems incorrect... I'd like to understand how/if these changes modify existing behavior where this method was used - it's important we don't try to load packages that already are loaded to avoid throwing errors that confuse users.

Reply:
There is no change in existing behavior. There was only one call internally to this utility in PackagePathViewModel and that was adapted accordingly.